registration/Dockerfile

```diff
@@ -1,32 +1,5 @@
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java17-22.1.0 AS builder
+FROM eclipse-temurin:17-jdk-alpine AS builder
 
-ADD . /build
-WORKDIR /build
-
-# For SDKMAN to work we need unzip & zip
-RUN yum install -y unzip zip
-
-RUN \
-    # Install SDKMAN
-    curl -s "https://get.sdkman.io" | bash; \
-    source "$HOME/.sdkman/bin/sdkman-init.sh"; \
-    sdk install maven; \
-    # Install GraalVM Native Image
-    gu install native-image;
-
-RUN source "$HOME/.sdkman/bin/sdkman-init.sh" && mvn --version
-
-RUN native-image --version
-
-RUN source "$HOME/.sdkman/bin/sdkman-init.sh" && mvn -B clean package -P native --no-transfer-progress
-
-
-FROM oraclelinux:7-slim
-
-RUN yum update -y && yum install -y curl
-
-# Add Spring Boot Native app pesto to Container
-COPY --from=builder "/build/target/pesto" pesto
-
-# Fire up our Spring Boot Native app by default
-CMD [ "sh", "-c", "./pesto -Dserver.port=$PORT" ]
\ No newline at end of file
+ADD target/pesto-registration.jar pesto-registration.jar
```

Comment:
no. don't do this. the jar file must NOT exists on the repository. build the jar file inside the dockerfile instead.

Reply:
done